torch/package/_stdlib.py

```diff
@@ -18,229 +18,8 @@ def is_stdlib_module(module: str) -> bool:
 
 def _get_stdlib_modules():
     if sys.version_info.major == 3:  # noqa: UP036
-        if sys.version_info.minor == 9:
```

Comment:
Add an asset about the minor version to silence the type error below.

Reply:
Funny thing is: none of those are/were really necessary, may be I don't even need an assert there, but let's keep it for now...